src/backends/tensorrt/plan_backend.cc

```diff
@@ -1673,7 +1673,13 @@ PlanBackend::InitializeGraphSpecs(
     // default build for batch sizes 1, 2, 3, 4, 6, 8, 12, 16, 'max_batch_size'.
     // If preferred batch size is specified, then the batch sizes will be
     // 1, preferred batch sizes, 'max_batch_size'.
-    std::set<int> cuda_graph_batch_sizes{1};
+    std::set<int> cuda_graph_batch_sizes;
```

Comment:
Are you seeing a failure because of the existing code?
A model with max_batch_size == 0 still needs to take a single request and in Triton that is equivalent to batch-size 1

Reply:
There is a check below to compare `batch_size <= max_batch_size`, so 1 will be ignored if max_batch_size == 0